src/project/server/routes/api.js

```diff
@@ -0,0 +1,114 @@
+// Define your endpoints here (this is your "controller file")
+const express = require('express');
+const ItemManager = require('../services/item-manager.js')
+
+const todoRouter = express.Router();
+const itemManager = new ItemManager();
+
+todoRouter.get('/', async(req, res) => {
+    const {sort, filter} = req.query;
+    
+    let sortedList = [];
+
+    const sortObj = {
+        atoz:{field: 'itemName', order:'ASC'},
+        ztoa:{field: 'itemName', order:'DESC'},
+        dtou:{field: 'status', order:'DESC'},
+        utod:{field: 'status', order:'ASC'},
+    }
+
+    if(sort){
```

Comment:
nesting like this makes for complex code that is hard to maintain, you should clean this up, either with separate endpoints or just more organzied logic where you check for something get the result and return it. 

The only way one controller would make sense to me is if you can have somethign that is sorted and filtered but in this case it seems that it's either one or the other or none, so one controller complicates things.